internal/ui/ime.go

```
@@ -0,0 +1,240 @@
+		return n, nil
+	}
+	col, row, ok := w.anchor.get()
+	if !ok {
+		return n, nil
+	}
+	_, err = io.WriteString(w.out, ansi.CursorPosition(col, row))
+	return n, err
+}
+
+func (w *cursorOutputWriter) trackAltScreen(p []byte) {
+	enter := bytes.LastIndex(p, []byte(ansi.SetAltScreenSaveCursorMode))
```

golangci-lint (2.12.2)

[error] SA1019: ansi.SetAltScreenSaveCursorMode is deprecated: use [ModeAltScreenSaveCursor] instead. 

(staticcheck)